terraform/aws/env/us-east/terraform.tfvars

```diff
@@ -1,7 +1,7 @@
-region            = "us-east-1"
-ami               = "ami-066a7f2ffac02f833"
-instance_type     = "t2.medium"
-key_name          = "KEY_NAME"
-server_count      = "3"
-client_count      = "4"
-
+region               = "us-east-1"
+ami                  = "ami-090a41df9e193a506"
```

Comment:
could we use a datasource to populate the default here? https://www.terraform.io/docs/providers/aws/d/ami.html

Reply:
good idea, but i'll leave that for the next person.